scripts/install_third_party.py

```diff
@@ -326,11 +328,37 @@ def download_manifest_files(filepath):
                     dependency_tar_root_name, dependency_target_root_name)
 
 
+def remove_emojis_from_guppy(guppy_file_path):
+    """The emoji symbols present in the guppy.min.js file are not compatible
+    with all platforms, hence they must be removed from the file.
```

Comment:
Mention that it it caused console errors due to this as well?

Reply:
Done.